models/aliyun_ftasr_api_zh/asr_sdk.py

```diff
@@ -122,6 +124,22 @@ def fileTrans(akId, akSecret, appKey, fileLink) :
         print ("录音文件识别失败！")
         return ""
 
+def asr(l):
+    l = l.strip()
+    if (len(l.split()) == 2): # scp format: "key\taudio"
+        key, audio = l.split(maxsplit=1)
+        sys.stderr.write('key:' + key + '\taudio:' + audio + '\n')
+        sys.stderr.flush()
+
+        text = ''
+        text = fileTrans(access_key_id, access_key_secret, APPKEY, audio)
+
+        trans.write(key + '\t' + text + '\n')
```

Comment:
* trans is out-of-scope of asr() ? I don't see asr() as a closure to capture that.
* do we need locks to avoid concurrency contention ? 

Please take your PR with care, always validate your submission.  It's impossible for me to look into the code of all model-images inside model-zoo, and each failing benchmarking run *WILL* cost time & money.